includes/sys_form.php

```diff
@@ -20,34 +20,32 @@ function form_hidden($name, $value)
  *
  * @param string $name
  * @param string $label
- * @param string $value
+ * @param float  $value
```

Comment:
You use a `number` field with a stepping of 1 here. So we should use `int` instead of `float`.

```suggestion
 * @param int  $value
```

Reply:
changed. thought about if we use this function somewhere else where the step size is a float that this is already prepared. but then we also need to change the function signature to provide the step size